### PR TITLE
Improve upgrading docs

### PR DESCRIPTION
**Pull request by rata**
Here are some small changes to the upgrading documentation that I found when doing following it myself. I took the liberty to do some small and simple improvements (like add the `set-image` example for kube-proxy).

I tried to split the logical changes in different commits and explain on each commit message, but if something is not clear please don't hesitate to ask.

Thanks a lot!
Rodrigo

### CODE REVIEW

**Comment by rmenn** [comment]
could you please squash the commits, please ?

**Comment by rata** [comment]
@rmenn sure, like this? If not, feel free to merge with the squash option in github and edit, if that is easier. Hopefully this is fine, though :)

File: Documentation/upgrading.md [hunk 1]
```diff
@@ -120,24 +126,18 @@ Verify the control plane components updated.
...
 
-```sh
-$ kubectl get nodes
-NAME                               STATUS    AGE       VERSION
-node1.example.com                  Ready     21d       v1.6.7+coreos.0
-node2.example.com                  Ready     21d       v1.6.7+coreos.0
-node3.example.com                  Ready     21d       v1.6.7+coreos.0
-node4.example.com                  Ready     21d       v1.6.7+coreos.0
-```
+Also, consider running `kubectl -n kube-syste get pods` to verify all upgraded pods are running correctly.
```
Comment (bradbeam) [inline Documentation/upgrading.md]: typo: `kube-syste` -> `kube-system`

Comment (rata) [reply Documentation/upgrading.md]: ouch! Thanks, fixed now :)

**Comment by rata** [comment]
@andrewrynhard @bradbeam fixed now, thanks! :-D

**Comment by rata** [comment]
@rmenn @andrewrynhard I think the GH approve status is not used by tide. I think you need to do /lgtm and /approve commands for tide to merge this (I used that on `kubernetes` org, I guess is the same :))

**Comment by andrewrynhard** [comment]
Hmm. Seems to think I’m not an owner.

**Comment by andrewrynhard** [comment]
Gonna give this another go hoping that the move to kubernetes-sigs fixed any issues.

/approve

**Comment by rata** [comment]
Thanks for all the attempts to merge this! :-D

**Pull request merged**